Kwf/Media/Output/ClearCacheInterface.php

```diff
@@ -0,0 +1,5 @@
+<?php
+interface Kwf_Media_Output_ClearCacheInterface
+{
+    public static function canCacheBeDeleted($id);
+}
```

Comment:
This is only the first example... currently I need this feature for a concrete usecase in a web. Also it should still be implemented for other components using media (I think download and video component, maybe some others...)